pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1175,6 +1179,10 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
           await this.vtun.start();
         }
 
+        const rdNetworking = !!config?.experimental.virtualMachine.networkingTunnel;
+
+        this.privilegedServiceEnabled = rdNetworking ? false : await this.invokePrivilegedService('start');
```

Comment:
We could potentially passed this to `installGuestAgent` (the call is later in this function) rather having state on the backend. Not sure how much we care though.